service/src/main/java/bio/terra/policy/service/region/RegionService.java

```diff
@@ -61,11 +72,47 @@ public Region getRegion(String name) {
     return regionNameMap.get(name);
   }
 
+  public HashSet<String> getPaoDatacenters(Pao pao, String platform) {
```

Comment:
Looks like you are not filtering by platform. Do you want that at this level?

I think either here or in the controller, I think we want just the datacenter names for the platform and not the <platform>.<datacenter> format. I would expect that in either the error from `evaluate` or `list-valid` result.

Reply:
I was debating where to do that filtering. Since we have the platform passed all the way to this level, it does make sense to do it here.